webapp/src/translations/languages/en.json

```diff
@@ -1001,11 +1001,11 @@
     "downloading": "Downloading",
     "applyingSnapshot": "Applying snapshot",
     "unpackingSnaphot": "Unpacking snapshot from %{address}",
-    "verifyingSnapshot": "Verifying snapshot...",
-    "installingSnapshot": "Installing snapshot...",
+    "verifyingSnapshot": "Verifying snapshot ...",
+    "installingSnapshot": "Installing snapshot ...",
     "snapshotApplied": "Snapshot applied",
     "startSyncBlock": "DeFi Wallet will now start synchronizing at block %{from}.",
-    "pleaseWait": "Please wait...",
+    "pleaseWait": "Please wait ...",
```

Comment:
"Please wait..." should remain as "Please wait...", no `space`.

Reply:
You are right. I did some research and removed the space before the ellipsis.